src/rest/subhandlers/rest_basic_queries.erl

```diff
@@ -40,9 +39,6 @@ init(Req, State) ->
 allowed_methods(Req, State) ->
     {[<<"POST">>], Req, State}.
 
-content_types_provided(Req, State) ->
-    {[{<<"application/json">>, to_json}], Req, State}.
-
 content_types_accepted(Req, State) ->
```

Comment:
Podobnie jak w przypadku `content_types_provided`
```suggestion
accepted_content_types(Req, State) ->
```

To też przewija się dość często, sprawdź też pozostałe handlery